notebook/static/notebook/js/codecell.js

```diff
@@ -385,7 +385,12 @@ define([
      * @private
      */
     CodeCell.prototype._handle_set_next_input = function (payload) {
-        var data = {'cell': this, 'text': payload.text, replace: payload.replace};
+        var data = {
+            cell: this,
```

Comment:
Yup, quotes in object literals is our Python habits leaking into our js. The result is the same, though.